### PR TITLE
Updated ClearLinux

### LINKED ISSUE #192: Clear Linux failing
Continues on error is being enable to prevent it from failing CI

### PR DESCRIPTION
**Pull request by kiazhi**
## PR Summary

- fixes #192 
- includes update of ClearLinux Image and PowerShell version

> Note:
> `gssntlmssp` is not included because there is no such bundle available, therefore
> GssNtlmSspTests is skipped in the `meta.json` file.
>
> The `meta.json` file included the below:
> `"SkipGssNtlmSspTests": true`

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [x] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/assets/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 1 hour

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 1 hour ago

File: release/community-stable/clearlinux/docker/Dockerfile [hunk 1]
```diff
@@ -4,69 +4,18 @@
...
-    && tar zxf /tmp/less.tar.gz -C /tmp/less --strip-components 1 \
-    # make less executable
-    && ./tmp/less/configure \
-    # create powershell folder
-    && mkdir -p ${PS_INSTALL_FOLDER} \
-    # uncompress powershell linux tar file
-    && tar zxf /tmp/powershell-linux.tar.gz -C ${PS_INSTALL_FOLDER}
-
-# Start a new stage so we lose all the tar.gz layers from the final image
-FROM ${imageRepo}:${fromTag}
```
Comment (TravisEz13) [inline release/community-stable/clearlinux/docker/Dockerfile:61]: We used multiphase to reduce size, but this is just an FYI...

File: release/community-stable/clearlinux/docker/Dockerfile [hunk 2]
```diff
@@ -4,69 +4,18 @@
...
 # Define Args for the needed to add the package
-ARG PS_VERSION=6.1.0
+ARG PS_VERSION=7.0.1
 ARG PS_PACKAGE=powershell-${PS_VERSION}-linux-x64.tar.gz
 ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
-ARG PS_INSTALL_VERSION=6
+ARG PS_INSTALL_VERSION=7
 
 # Download the PowerShell Core Linux tar.gz and save it
 ADD ${PS_PACKAGE_URL} /tmp/powershell-linux.tar.gz
```
Comment (TravisEz13) [inline release/community-stable/clearlinux/docker/Dockerfile:16]: More FYI, This will be in the final image because this layer is committed at this point.

**Pull request merged**